cypher/cypher-docs/src/docs/dev/ql/constraints/index.asciidoc

```diff
@@ -133,6 +132,6 @@ include::create-a-node-that-complies-with-a-node-key.asciidoc[leveloffset=+1]
 
 include::create-a-node-that-violates-a-node-key.asciidoc[leveloffset=+1]
 
-//include::removing-a-node-key-constrained-property.asciidoc[leveloffset=+1]
+include::removing-a-node-key-constrained-property.asciidoc[leveloffset=+1]
```

Comment:
How does this differ from `Drop a Node Key` above?

Reply:
OK. I think I found it below. So this is not about removing the NODE KEY, but what happens when you remove a property that is constrained. Makes sense now.

Reply:
👍